test/quantization/core/test_quantized_op.py

```diff
@@ -3661,6 +3661,26 @@ def test_embedding_bag_2d_indices(self):
         torch.testing.assert_close(result, qresult, atol=0.05, rtol=1e-3)
 
 
+class TestDynamicQuantizedConv(TestCase):
+    def test_qconv(self):
+        X_fp32 = torch.randn(3, 3, 4, 4)
+        reduce_range = True
+        s, z = _calculate_dynamic_qparams(X_fp32, torch.quint8, reduce_range)
+        X_q = torch.quantize_per_tensor(X_fp32, s, z, torch.quint8)
```

Comment:
I think general practice has been to quantize the tensor and then dequantize to get `X_fp32` due to loss in precision if you directly operator on original X_fp32. Check other tests in the file.

Reply:
Also maybe have more test cases generated similar to other tests?

Reply:
okay, will do